sky/engine/core/script/dart_init.cc

```diff
@@ -71,6 +71,7 @@ static const char* kDartProfilingArgs[] = {
     // default profile period to 100Hz. This number is suitable for older
     // Raspberry Pi devices but quite low for current smartphones.
     "--profile_period=1000",
+    "--conditional_directives",
```

Comment:
This ends up in `kDartProfilingArgs`. Not sure conditional_directives are related to profiling.